docs/source/addons/i18n.md

```diff
@@ -3,66 +3,30 @@ html_meta:
   "description": ""
   "property=og:description": ""
   "property=og:title": ""
-  "keywords": ""
+  "keywords": "Internationalization, i18n, add-on"
 ---
 
 # Add-on Internationalization
 
-Volto added add-on internationalization support since Volto 11.1.0 via an enhanced `src/i18n.js` script.
-In case you can't upgrade to this version, you can still support it in your project, by backporting
-it and add it into your project's workflow.
+The internationalization workflow is the same as in main Volto, you develop your add-on, add the translations to your code. 
+See {ref}`creating-i18n-strings` how to mark strings and phrases as translatable.
 
-1. Copy over the updated https://github.com/plone/volto/blob/master/src/i18n.js script to your project, e.g `src/i18n.js`
+Your add-on has a `locales` folder with a `.pot` file.
 
-2. Update your project's `package.json` to call it, instead of calling the inner Volto one.
+1. Create the following structure in your add-ons `locales` folder for every language you want to support. As an example the language italian:
 
-```json
-  "scripts": {
-...
-    "i18n": "NODE_ENV=production node src/i18n.js",
-...
-  }
+```text
+it
+└── LC_MESSAGES
+    └── volto.po
 ```
```

Comment:
Indent the above block so that it falls within the first enumerated list item. I have not checked, but I think it might break subsequent numbering.